docs/installation/linux.txt

```diff
@@ -46,7 +46,7 @@ Scroll further below for more OS-specific insructinos.
     cd bigartm
     mkdir build && cd build
     cmake ..
-    make
+    make -j20
```

Comment:
@akashin I would keep this as is. Here is another project that prefer to keep instructions as short as possible: https://github.com/google/glog . I would say on Linux users are themselves responsible to run make in parallel.